src/sympc/protocol/beaver/beaver.py

```diff
@@ -14,24 +14,33 @@ def build_triples(x, y, op_str):
     if op_str not in EXPECTED_OPS:
         raise ValueError(f"{op_str} should be in {EXPECTED_OPS}")
 
-    session = x.session
+    from sympc.tensor import AdditiveSharingTensor
+    from sympc.tensor import FixedPrecisionTensor
```

Comment:
maybe avoid imports inside functions. 
When we couldn't do this, sometimes we could import let's say `syft` and then call `syft. FixedPrecisionTensor`

Reply:
Need to handle this